integration-tests/pom.xml

```diff
@@ -48,6 +48,19 @@
         <module>jaxrs</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <id>jdk11-integration-test-modules</id>
+            <activation>
+                <jdk>[11,)</jdk>
+            </activation>
+            <modules>
+                <module>jakarta-ee-support</module>
+                <module>jakarta-ee</module>
+            </modules>
+        </profile>
+    </profiles>
+
```

Comment:
Maybe we should just require Java 11 anyway… Didn't we agree on this on the ML?

Reply:
you are right, but since it hasn't been implemented yet, this needs to stay for the tests to pass, for now.

Reply:
sync'd up with jdk 11+ changes!